.github/workflows/ci.yml

```diff
@@ -12,23 +12,19 @@ jobs:
     strategy:
       matrix:
         go:
-          - '1.18'
+          - '1.19'
     steps:
       - uses: actions/checkout@v3
-      - name: Submodules
-        run: git submodule update --init --recursive
-      - name: golangci-lint
-        uses: golangci/golangci-lint-action@v3
```

Comment:
why did you remove this action? it installs golangci-lint.

Reply:
Because its also run from sh script. And I'd prefer to run it in sh script, cause we can add other linters in that script (like in caminogo)